akka-cluster-typed/src/test/scala/akka/cluster/typed/ClusterSingletonApiSpec.scala

```diff
@@ -123,12 +123,13 @@ class ClusterSingletonApiSpec extends ScalaTestWithActorTestKit(ClusterSingleton
       val cs2 = ClusterSingleton(adaptedSystem2)
 
       val settings = ClusterSingletonSettings(system).withRole("singleton")
-      val node1ref = cs1.spawn(pingPong, "ping-pong", Props.empty, settings, Perish)
-      val node2ref = cs2.spawn(pingPong, "ping-pong", Props.empty, settings, Perish)
+      val singleton = SingletonActor(pingPong, "ping-poing").withStopMessage(Perish).withSettings(settings)
```

Comment:
Was `ping-poing` intentional? Should that cause a fail?

Reply:
It was not! But it doesn't make any difference for the test as it was testing that if you pass in the same SingletonActor you get back the same reference back so doesn't matter what name it is

Reply:
However it is a better test if SingletonActor is inclined to make sure reference equality of it isn't used and it is the name